Makefile

```diff
@@ -6,7 +6,7 @@ unexport GOFLAGS
 # but otherwise can be installed elsewhere on developers machines
 BASE_DIR=$(shell pwd)
 export PATH:=$(BASE_DIR)/bin:$(PATH)
-SHELL := env PATH=$(PATH) /bin/bash
+SHELL := /bin/bash
```

Comment:
Please take note of this. I don't think this will impact anything, but it's probably the the most significant change in this PR.

Reply:
If I understand this correctly, the original line is trying to reset the env to use the actual user $PATH to then pick the bash interpreter.  I *think* this is fine.

Reply:
Yeah, I would like to keep this how it was but since installing JetBrains Toolbox, I now have a space in my PATH that breaks this line as is. I'm no Makefile expert, but this worked for me.